src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/MessageHeaderTest.java

```diff
@@ -23,43 +21,55 @@ void testValidateSequenceNumberNullThrowsException() {
 
     @Test
     void testValidateSequenceNumberLessThanMinimumValueThrowsException() {
-        final MessageHeader messageHeader = new MessageHeader(0L);
+        final MessageHeader messageHeader = MessageHeader.builder()
+            .sequenceNumber(0L)
+            .messageType(MessageType.PATHOLOGY)
+            .build();
 
         final EdifactValidationException exception = assertThrows(EdifactValidationException.class,
                 messageHeader::validate);
 
         assertEquals("UNH: Attribute sequenceNumber must be between 1 and 99999999", exception.getMessage());
     }
 
+    @SuppressWarnings("checkstyle:MagicNumber")
```

Comment:
Easier to put this on the whole class. As you know, I don't think magic numbers should even be flagged as potentially problematic in test code.

Reply:
True, but rather just be explicit on each test. Seems checkstyle for magicnumber checks tests as well, doesn't look like there's an option to exclude the `/tests` folder as well